.github/workflows/ci.yml

```diff
@@ -92,7 +92,7 @@ jobs:
 
           - name: ubuntu:production-dbg
             os: ubuntu-20.04
-            config: production --auto-download --assertions --tracing --unit-testing --all-bindings --editline --cocoa
+            config: production --auto-download --assertions --tracing --unit-testing --all-bindings --editline --cocoa --gpl
```

Comment:
This seems like the right fix; let me know if it's not.

Reply:
The right fix for CI. I guess we don't want to auto-enable `--gpl` at configure time if `--cocoa` is enabled. But we should tell the user that this is the reason why it fails and suggest to enable `--gpl`, if possible. ATM the message output by CMake is a bit cryptic. Can you try to catch this in the configure script?

Reply:
Yeah, improving the error message is a good idea.
I did it in `CMakeLists.txt` to avoid spreading configuration validation code across two files.

Reply:
Nice!